packages/backend/server/src/models/workspace.ts

```diff
@@ -58,6 +58,13 @@ export class WorkspaceModel extends BaseModel {
     this.logger.debug(
       `Updated workspace ${workspaceId} with data ${JSON.stringify(data)}`
     );
+
+    if (workspace.enableDocEmbedding) {
```

Comment:
A 'workspace.updated' event should be added, and then this event should be listened to for AI embedding logic to avoid the logic of the workspace model being aware of AI

Reply:
done